homeassistant/components/media_player/__init__.py

```diff
@@ -889,6 +895,8 @@ def _async_fetch_image(hass, url):
         if response.status == 200:
             content = yield from response.read()
             content_type = response.headers.get(CONTENT_TYPE_HEADER)
+        elif response.status in (400, 404):
+            content_type = True
```

Comment:
I think it would be better just to add a third return variable rather than reusing content_type for a different purpose.